regression/check_style.sh

```diff
@@ -124,8 +124,16 @@ echo " "
 
 patchfile_c=$(mktemp /tmp/gcf.patch.XXXXXXXX)
 
-# don't actually modify the files (compare to branch 'develop')
-cmd="${gcf} --binary ${cf} -f --diff --extensions hh,cc,cu develop"
+# don't actually modify the files (originally we compared to branch 'develop',
+# but let's try ORIG_HEAD or maybe use CI variables like TRAVIS_BRANCH or
+# CI_MERGE_REQUEST_TARGET_BRANCH_NAME.
+target_branch=ORIG_HEAD
+if [[ -n ${TRAVIS_BRANCH} ]]; then
+  target_branch=${TRAVIS_BRANCH}
+elif [[ -n ${CI_MERGE_REQUEST_TARGET_BRANCH_NAME} ]]; then
```

Comment:
* Forgot to send these variables to the docker image.  I'll need to update this PR.
* Also, git doesn't seem to know `ORIG_HEAD`.  I'll need to investigate that also.